config/locales/en.yml

```diff
@@ -14,6 +14,11 @@ en:
     attributes:
       trainee:
         trainee_id: "Trainee ID"
+    validations:
+      trainee:
+        nationalities:
+          empty: Please select at least one nationality
+          format: Please enter your date of birth in the correct format
```

Comment:
I think we have to avoid using 'please', at least according to guidance https://design-system.service.gov.uk/components/error-message/

Reply:
Yeah good call. Although I'm not really sure what copy needs to be here as the ticket didn't specify.

Reply:
How would we like to cover these? We might be a bit premature to fully finalise messages - but it would also be good to set up some sensible defaults. 

The design system has [guidance on suggested messages](https://design-system.service.gov.uk/components/text-input/#error-messages) as part of each pattern. I suspect 90% of the time we'll base ours on these patterns. 

If possible to automate in that format it would be great. I'm not sure this would be fully possible as the target (you / they) changes.

For most inputs I anticipate at least 4 messages:
* Empty / not provided
* Too short
* Too long
* Invalid in some other way (wrong characters, etc)

For most checkboxes / radios:
* Empty / not provided
* Provided answer is invalid

For conditional stuff:
* the error is a bit special as the user either has to answer the conditional *or* select a different parent choice.

Reply:
@JonF1 done now